dev/scripts/provision-elasticsearch.sh

```diff
@@ -0,0 +1,17 @@
+add-apt-repository ppa:openjdk-r/ppa
```

Comment:
Yeah makes sense.  I was just doing copy-pasta based on the surrounding code for postgres and chef-backend to make my life easier in the Vagrantfile.